dg-core/src/main/java/org/finra/datagenerator/distributor/multithreaded/SearchWorker.java

```diff
@@ -52,7 +53,11 @@ public SearchWorker(final Frontier frontier, final Queue<Map<String, String>> qu
     @Override
     public void run() {
         log.info(Thread.currentThread().getName() + " is starting DFS");
-        frontier.searchForScenarios(queue, flag);
+        try {
+            frontier.searchForScenarios(queue, flag);
+        } catch (IOException e) {
+            e.printStackTrace();
```

Comment:
Don't use printStackTrace() and instead write to the logger using log.error("<Appropriate message>",e);